Drop pkgutil-style namespace to prevent collision with other smpp.*
packages (e.g., smpp.twisted) and specify the package name/path
explicitly.

1. https://projects.gentoo.org/python/guide/concept.html#adding-new-namespace-packages-to-gentoo
2. https://stackoverflow.com/questions/54430694/python-setup-py-how-to-get-find-packages-to-identify-packages-in-subdirectori
--- a/setup.py
+++ b/setup.py
@@ -9,7 +9,11 @@
     author_email = "roger.hoover@gmail.com",
     description = "Library for parsing Protocol Data Units (PDUs) in SMPP protocol",
     license = 'Apache License 2.0',
-    packages = find_packages(exclude=["tests"]),
+    packages = ["smpp.pdu"],
+    package_dir = {
+        "": ".",
+        "smpp.pdu": "smpp/pdu",
+    },
     keywords = "smpp pdu",
     url = "https://github.com/mozes/smpp.pdu",
     py_modules=["smpp.pdu"],
--- a/smpp/__init__.py
+++ /dev/null
@@ -1,16 +0,0 @@
-"""
-Copyright 2009-2010 Mozes, Inc.
-
-   Licensed under the Apache License, Version 2.0 (the "License");
-   you may not use this file except in compliance with the License.
-   You may obtain a copy of the License at
-
-       http://www.apache.org/licenses/LICENSE-2.0
-
-   Unless required by applicable law or agreed to in writing, software
-   distributed under the License is distributed on an "AS IS" BASIS,
-   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either expressed or implied.
-   See the License for the specific language governing permissions and
-   limitations under the License.
-"""
-__path__ = __import__('pkgutil').extend_path(__path__, __name__)
\ No newline at end of file
